todo_app/tests/test_view_model.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from todo_app.classes.card import Card
+from todo_app.classes.view_model import ViewModel
+
+# Arrange
+@pytest.fixture
+def test_view_model():
+    test_cards = []
+    test_cards.append(Card(1, "test1", "To Do"))
+    test_cards.append(Card(2, "test2", "Doing"))
+    test_cards.append(Card(3, "test3", "Doing"))
+    test_cards.append(Card(4, "test4", "Done"))
+    test_cards.append(Card(5, "test5", "Done"))
+    test_cards.append(Card(6, "test6", "Done"))
+
+    return ViewModel(test_cards)
+
+def test_view_model_done_property(test_view_model):
+
+# Act
+    result = test_view_model.done_cards
+
+# Assert
+    assert len(result) == 3
+    assert result[0].name == "test4"
```

Comment:
Out of curiosity, _do_ you care whether "test4" is the _first_ entry in the returned list? If someone restructured the `done_cards` property so that it wasn't guaranteed to follow the same order as the underlying items list, would that be wrong?

In our case, the task is sufficiently undefined that the answer is whatever you like, but worth noting that this test implicitly claims that that restructure would be wrong!

Reply:
Good point, thanks!